sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1233,7 +1230,12 @@ public Long countCasesForMap(
 		Predicate nameSimilarityFilter =
 			cb.gt(cb.function("similarity", double.class, nameSimilarityExpr, nameSimilarityExpr2), configFacade.getNameSimilarityThreshold());
 		Predicate diseaseFilter = cb.equal(root.get(Case.DISEASE), root2.get(Case.DISEASE));
-		Predicate regionFilter = cb.equal(region.get(Region.ID), region2.get(Region.ID));
+		Predicate responsibleRegionFilter = cb.or(
```

Comment:
Okay, after seeing this code, I think it actually doesn't make sense to do a cross-check here. We should only check whether responsible jurisdiction == responsible jurisdiction and place of stay == place of stay when searching for duplicates.